service-worker.tmpl

```diff
@@ -34,10 +34,8 @@
 importScripts(<%= importScripts %>);
 <% } %>
 
-/* eslint-disable quotes, comma-spacing */
 var PrecacheConfig = <%= precacheConfig %>;
-/* eslint-enable quotes, comma-spacing */
-var CacheNamePrefix = 'sw-precache-<%= version %>-<%= cacheId %>-' + (self.registration ? self.registration.scope : '') + '-';
+var CacheName = 'sw-precache-<%= version %>-<%= cacheId %>-' + (self.registration ? self.registration.scope : '');
```

Comment:
Oh, I see that's not being followed throughout the file actually, so I guess this is actually fitting in with the surrounding code.